db/seeds.rb

```diff
@@ -5,37 +5,47 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
-User.create!(email: 'Test@Fake.com', password: 'password123', sign_in_count: 0)
+if Rails.env.development?
+  if User.find_by(email: 'Test@Fake.com').nil?
+    User.create(email: 'Test@Fake.com', password: 'password123')
+  end
```

Comment:
We want this script to be "exceptional", by utilizing `.create!` (or any other administrative input settings, rather than user input):

```ruby
User.find_or_create_by!({ email: 'Test@Fake.com' }, { password: 'password123' })
```